SwitchSpeak-iOS/SwitchSpeak-iOS/TouchGrid/Tree/Node.swift

```diff
@@ -47,7 +47,31 @@ class Node {
 		for childNode in self.childNodes {
 			childNode.unHighlightSubTree()
 		}
-	}	
+	}
+    
+    // it will retrun a node. if it has only undummy child node, it will return that child node, else it will return itself
+    func mergeNode() -> Node {
+        var unDummyNum = 0
+        var unDummyChildren = Node()
+        for childNode in self.childNodes {
+            if childNode.dummy == false {
+                unDummyNum = unDummyNum + 1
+                unDummyChildren = childNode
+            }
+        }
+        if unDummyNum == 1 {
+            return unDummyChildren
+        }
+        else{
```

Comment:
Could you please make sure whenever you put a curly brace that you have a space behind it, and put the else statement on the same line as the closing brace of the if statement?
```swift
if nonDummyChildCount {
    return nonDummyChild
} else {
     return self
}